src/js/footer/disclaimerAddClassname/index.js

```diff
@@ -1,9 +1,7 @@
-document.addEventListener("DOMContentLoaded", function () {
```

Comment:
After we update the script in webflow, the JS files will be loaded after webflow site is completely loaded.